src/propodesolvers.jl

```diff
@@ -0,0 +1,34 @@
+abstract QuODESolvers <: QuPropagatorMethod
+
+const type_to_method = @compat Dict{Any, Any}(:QuODE45 => ODE.ode45, :QuODE78 => ODE.ode78, :QuODE23s => ODE.ode23s)
+
+for op in keys(type_to_method)
+    @eval  begin
+        @doc """
+        ODE Methods types
```

Comment:
Does this lead to a duplication of the documentation? Maybe you could at least add the type name to the docstring?